src/components/Avatar/Avatar.tsx

```diff
@@ -39,6 +39,7 @@ const StyledRoot = styled(Root)`
     background-color: ${props =>
       props.theme.click.avatar.color.background.hover};
     color: ${props => props.theme.click.avatar.color.text.hover};
+    cursor: pointer;
```

Comment:
Should we force it for avatar
Is there a possibility for the avatar be shown without a link and if in case of link then it would automatically add the cursor

Reply:
Yeah good point, makes no sense in the user table for example.